beets/util/functemplate.py

```diff
@@ -512,6 +513,18 @@ def _parse_ident(self):
         return ident
 
 
+class ArgumentsParser(Parser):
+    """``Parser`` that considers ``ARG_SEP`` to be a special character.
```

Comment:
Perhaps a more descriptive docstring would say "a parser used inside function arguments," i.e., state the purpose first before getting into the implementation details.